lib/tools.js

```diff
@@ -2379,6 +2379,23 @@ function parseShortGithubUrl(url) {
     };
 }
 
+/**
+ * Removes properties which are given by preserve
+ * @param {object} preserve - object which has true entries for all attributes which should be removed from currObj
+ * @param {object} oldObj - old object
+ * @param {object} newObj - new object
```

Comment:
On a side note, `object` is equivalent to `any`

Reply:
afaik there is no real alternative without defining each attribute of the object separately

Reply:
besides ts ;-)

Reply:
`Record<string, any>`

Reply:
won't help much in this case though

Reply:
I don’t find a valid documentation of Record for plain jsdoc but would be interesting for future implementation s